openflow13/multipart_test.go

```diff
@@ -0,0 +1,142 @@
+package openflow13
+
+import (
+	"bytes"
+	"testing"
+
+	"antrea.io/libOpenflow/common"
+)
+
+func TestMultipartMessage(t *testing.T) {
+	feature := newTableFeatures()
+	mpartRequest := &MultipartRequest{
+		Header: NewOfp13Header(),
+		Type:   MultipartType_TableFeatures,
+		Flags:  0,
+		Body:   feature,
+	}
+	reqBytes, err := mpartRequest.MarshalBinary()
+	if err != nil {
+		t.Fatalf("function MarshalBinary failed: %v", err)
+	}
+	deReq := new(MultipartRequest)
+	err = deReq.UnmarshalBinary(reqBytes)
+	if err != nil {
+		t.Fatalf("function UnmarshalBinary failed: %v", err)
+	}
+	if !mpartRequestEquals(mpartRequest, deReq) {
+		t.Errorf("Original MultipartRequest not equal to the decoded object")
+	}
```

Comment:
can we start using the `assert` and `require` assertions from testify in these unit tests?